src/core/types.ts

```diff
@@ -23,6 +23,7 @@ export interface TestOptions {
     launch?: LaunchOptions
   }
   server: boolean
+  endpoint?: string
```

Comment:
suggestion: rename `endpoint` to `host` to sync with Vite (and other) commonly named settings